{{cookiecutter.repostory_name}}/.github/workflows/cd.yml

```diff
@@ -22,4 +22,7 @@ jobs:
           ./deploy-to-aws.sh $ENVIRONMENT
           export TAG=deployed-${ENVIRONMENT}-`date -u +"%Y-%m-%dT%H.%M.%S"`
           git tag $TAG
-          git push origin $TAG
\ No newline at end of file
+          git push origin $TAG
+        env:
+          commit_msg : {% raw %} ${{ github.event.head_commit.message }} {% endraw %}
```

Comment:
Could change that for list of all commit messages since last deploy but need to do some testing for that

Reply:
we need to see all commits that have been deployed